internal/service/workspacesweb/testdata/tmpl/identity_provider_tags.gtpl

```diff
@@ -0,0 +1,16 @@
+resource "aws_workspacesweb_portal" "test" {
+  display_name = "test"
+}
+
+resource "aws_workspacesweb_identity_provider" "test" {
```

Comment:
nit - the resource which is the subject of the generated test should appear first in the template.